backend/orders/views.py

```diff
@@ -3,17 +3,39 @@
 from rest_framework import status
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from orders.models import Order
-from orders.serializers import OrderSerializer
+from .models import Order
+from .serializers import OrderSerializer
 
 
 @api_view(['GET', 'POST'])
 def order_list(request, format=None):
     """
-    List all code orders, or create a new order.
+    List all code orders, filter orders, or create a new order.
     """
+    token = request.GET.get('token')
+    if token is None or token == '':
```

Comment:
By the way, usually token is placed in the header section of the request. If you have a time, try to transfer it such a way. If you have no time it is okay for MVP to leave that as is. Just create issue with low priority to not forget about this fact